build-scripts/gulp/entry-html.js

```diff
@@ -92,13 +90,11 @@ const minifyHtml = (content, ext) => {
 };
 
 // Function to generate a dev task for each project's configuration
-// Note Currently WDS paths are hard-coded to only work for app
 const genPagesDevTask =
```

Comment:
https://github.com/home-assistant/frontend/blob/bda0642f02fab01755f131da9e6dce09c30a472a/build-scripts/gulp/entry-html.js#L290

Should be updated, remove the undefined param